neo/Consensus/ConsensusService.cs

```diff
@@ -175,8 +175,21 @@ private void InitializeConsensus(byte viewNumber)
 
         private void Log(string message, LogLevel level = LogLevel.Info)
         {
+            var currentColor = new ConsoleColorSet();
```

Comment:
Only thing I don't like here is this Dynamic memory criation for object... couldn't this all be static? it's such a simple thing...

Reply:
If you want, we could store the initial color in a static var